assets/src/edit-story/components/form/button.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+const Button = styled.button`
+  color: ${({ theme }) => theme.colors.mg.v3};
```

Comment:
There's a foreground, but not background color?

Reply:
When you say foreground you say the text color @dvoytenko declaration? Just asking to be more in sync with the PR as I and @ndev1991 will take it to work 😄 Also, just to be clear, what you are missing here is the background-color based on theme support, right?

Reply:
To clarify, this is in relation to the fact that it has a constant text color but no background color so it takes the background color of its container meaning that there could be potentially be a case where the text and background are both dark. Since I only expect this to be used for the design panel for now I added a white background color.